### PR TITLE
 Added a work-around to keep the remember_web_ cookie

### PR DESCRIPTION
**Pull request by Livijn**
Basically, it just stores the cookie in a session when you impersonate, then it restores the cookie when you leave the impersonation.

Solves #31

### CODE REVIEW

**Comment by drbyte** [comment]
yay! This worked as desired on my local test site.

(formerly my remember token wasn't functional after ending impersonation, which left me logged out after my session expired. Now I'm not left logged out anymore!)

**Comment by Livijn** [comment]
@MarceauKa mind merging this?

**Comment by MarceauKa** [comment]
Its BC, I'll merge with a new version

**Comment by MarceauKa** [comment]
Why PHP 7.2, Laravel 5.8 requires PHP 7.1.3.

File: src/Services/ImpersonateManager.php [hunk 1]
```diff
@@ -92,9 +96,12 @@ public function leave()
             $this->app['auth']->quietLogout();
             $this->app['auth']->quietLogin($impersonator);
 
+            $this->extractAuthCookieFromSession();
+
             $this->clear();
 
         } catch (\Exception $e) {
+            dump($e->getMessage());
```
Comment (drbyte) [inline src/Services/ImpersonateManager.php]: Was this `dump()` command supposed to be left in?

Comment (Livijn) [reply src/Services/ImpersonateManager.php]: Nope 👍

**Comment by Livijn** [comment]
> Why PHP 7.2, Laravel 5.8 requires PHP 7.1.3.

I'm not sure actually. I think it had to do with PHPUnit 8 and some version collisions.

**Comment by MarceauKa** [comment]
Rollback your PHP version changes and I merge it :)

**Pull request merged**